Gems/ScriptCanvas/Code/Editor/Framework/Interpreter.cpp

```diff
@@ -50,14 +50,9 @@ namespace ScriptCanvasEditor
             {
                 MutexLock lock(m_mutex);
 
-                if (m_runtimePropertiesDirty || !m_executor.IsPure())
+                if (m_runtimePropertiesDirty || !m_executor.IsPure() && m_executor.GetRuntimeOverrides().m_runtimeAsset.IsReady())
```

Comment:
do you need to put a bracket around `m_runtimePropertiesDirty || !m_executor.IsPure()`? or `&&` will be calculated first

Reply:
good catch, even if just for reading clarity I want the paranthesis.